types/ember-changeset-validations/index.d.ts

```diff
@@ -1,7 +1,15 @@
 import { ValidatorFunc } from 'ember-changeset/types';
 import { validator } from 'ember-validations';
 
-export type ValidatorFunction = (key: string, result: any) => string | string[] | true;
+export type ValidatorResult = string | string[] | true;
+
+export type ValidatorFunction = (
+    key: string,
+    newValue: undefined | unknown,
+    oldValue: undefined | unknown,
```

Comment:
`undefined | unknown` is the same as just `unknown`

Reply:
Lol, yep!